lib/routes/signin-codes.js

```diff
@@ -51,6 +51,13 @@ module.exports = (log, db, customs) => {
           return db.consumeSigninCode(code)
             .then(result => {
               return request.emitMetricsEvent('signinCode.consumed')
+                .then(() => {
+                  if (result.flowId) {
+                    return request.emitMetricsEvent('flow.continued', {
```

Comment:
No problem here, just scribbling my mad thoughts: I wonder if it'd make sense to have a `request.emitMetricsEvents(array)`. Maybe it could save extra memcached lookups, and be more concise in these cases...